### PR TITLE
gh-104773: PEP 594: Remove the aifc module

### PR DESCRIPTION
**Pull request by fatelei**
Remove aifc module 
issue #104773 

* Issue: gh-104773

----
:books: Documentation preview :books:: https://cpython-previews--104927.org.readthedocs.build/

### CODE REVIEW

File: Doc/whatsnew/3.11.rst [hunk 1]
```diff
@@ -1731,7 +1731,7 @@ Modules
   slated for removal in Python 3.13:
 
   +---------------------+---------------------+---------------------+---------------------+---------------------+
-  | :mod:`aifc`         | :mod:`chunk`        | :mod:`!msilib`      | :mod:`!pipes`       | :mod:`!telnetlib`   |
+  | :mod:`!aifc`         | :mod:`chunk`        | :mod:`!msilib`      | :mod:`!pipes`       | :mod:`!telnetlib`   |
```
Comment (DBJim) [inline Doc/whatsnew/3.11.rst:1734]:
```suggestion
  | :mod:`!aifc`        | :mod:`chunk`        | :mod:`!msilib`      | :mod:`!pipes`       | :mod:`!telnetlib`   |
```

**Review by DBJim — COMMENTED** [review]
Looks like the cause of docs test failure

File: .gitattributes [hunk 2]
```diff
@@ -1,6 +1,5 @@
 # Binary data types
 *.aif binary
-*.aifc binary
```
Comment (vstinner) [inline .gitattributes:3]: I don't think that this change is correct. The ``Lib/test/test_email/data/sndhdr.aifc`` file should still be treated as binary by Git.

**Review by vstinner — COMMENTED** [review]
I wrote a more complete PR: PR #104933.

**Comment by vstinner** [comment]
Thanks for your contribution! I merged my PR #104933 instead (more complete).

**Pull request closed without merging**